openml/extensions/sklearn/flow_functions.py

```diff
@@ -474,15 +491,16 @@ def _get_external_version_string(model, sub_components):
     for visitee in sub_components.values():
         for external_version in visitee.external_version.split(','):
             external_versions.add(external_version)
-    external_versions = list(sorted(external_versions))
-    external_version = ','.join(external_versions)
-    return external_version
+    return ','.join(list(sorted(external_versions)))
```

Comment:
For future PR: this string is not json parsable.(let's open an issue?)

Reply:
Done, it's #658.